include/NovelRT/Graphics/Vulkan/Utilities/PipelineBlendFactor.h

```diff
@@ -10,71 +10,38 @@
 
 namespace NovelRT::Graphics::Vulkan::Utilities
 {
-    [[nodiscard]] inline VkBlendFactor GetVulkanBlendFactor(GraphicsPipelineBlendFactor blendFactor)
+    static const VkBlendFactor NovelRTToVulkanBlend[] = {
+        VK_BLEND_FACTOR_ZERO,                     // Zero = 0,
+        VK_BLEND_FACTOR_ONE,                      // One = 1,
+        VK_BLEND_FACTOR_SRC_COLOR,                // SrcColout = 2,
+        VK_BLEND_FACTOR_ONE_MINUS_SRC_COLOR,      // OneMinusSrcColour = 3,
+        VK_BLEND_FACTOR_DST_COLOR,                // DstColour = 4,
+        VK_BLEND_FACTOR_ONE_MINUS_DST_COLOR,      // OneMinusDstColour = 5,
+        VK_BLEND_FACTOR_SRC_ALPHA,                // SrcAlpha = 6,
+        VK_BLEND_FACTOR_ONE_MINUS_SRC_ALPHA,      // OneMinusSrcAlpha = 7,
+        VK_BLEND_FACTOR_DST_ALPHA,                // DstAlpha = 8,
+        VK_BLEND_FACTOR_ONE_MINUS_DST_ALPHA,      // OneMinusDstAlpha = 9,
+        VK_BLEND_FACTOR_CONSTANT_COLOR,           // ConstantColour = 10,
+        VK_BLEND_FACTOR_ONE_MINUS_CONSTANT_COLOR, // OneMinusConstantColour = 11,
+        VK_BLEND_FACTOR_CONSTANT_ALPHA,           // ConstantAlpha = 12,
+        VK_BLEND_FACTOR_ONE_MINUS_CONSTANT_ALPHA, // OneMinusConstantAlpha = 13,
+        VK_BLEND_FACTOR_SRC_ALPHA_SATURATE,       // SrcAlphaSaturate = 14,
+        VK_BLEND_FACTOR_SRC1_COLOR,               // Src1Colour = 15,
+        VK_BLEND_FACTOR_ONE_MINUS_SRC1_COLOR,     // OneMinusSrc1Colour = 16,
+        VK_BLEND_FACTOR_SRC1_ALPHA,               // Src1Alpha = 17,
+        VK_BLEND_FACTOR_ONE_MINUS_SRC1_ALPHA,     // OneMinusSrc1Alpha = 18,
+    };
```

Comment:
Is a C style array the best option for this?

I don't really mind too much it's just kind of odd

Reply:
Seems that c style arrays are what people use for lookup tables. Haven't been able to find an example that uses something else.